src/java/org/apache/cassandra/config/GuardrailsOptions.java

```diff
@@ -181,6 +185,45 @@ public void setColumnsPerTableThreshold(int warn, int fail)
                                   x -> config.columns_per_table_fail_threshold = x);
     }
 
+    @Override
+    public Set<String> getKeyspacePropertiesWarned()
+    {
+        return config.keyspace_properties_warned;
+    }
+    
+    public void setKeyspacePropertiesWarned(Set<String> properties)
+    {
+        updatePropertyWithLogging("keyspace_properties_warned",
+                                validateKeyspaceProperties(properties, "keyspace_properties_warned"),
+                                () -> config.keyspace_properties_warned, x -> config.keyspace_properties_warned = x);
```

Comment:
nit: Here and in the methods below, let's align the arguments horizontally. For example...
```suggestion
                                 () -> config.keyspace_properties_warned, x -> config.keyspace_properties_warned = x);
```